lib/sequel/adapters/drill.rb

```diff
@@ -84,16 +84,16 @@ class Dataset < Sequel::Dataset
       GREATER_THAN = '>'.freeze
       
       def fetch_rows(sql)
+        # hacks for Sequel functions without adapter methods intended to be extended/overridden
+        
+        # aggregate functions should include backticks
+        # TODO: check for safety/alternatives to stripping quotation marks as done below
+        sql = sql.gsub!('"',"")
```

Comment:
It's been there all along, I just relocated it. It is necessary because Sequel generates queries like:

`SELECT "something" FROM "something"`

and we don't want this converted to:

`SELECT "something" FROM dfs.tmp.`"something"`

I admit that this is one of the first changes I made originally intended as a quick workaround for deeper experimentation and I left a stub for myself (as you can see) to verify that this won't cause unintended problems. So far it hasn't. Can you think of any use cases where we might want to include quotes in queries? One thing I can think of is quoting strings in expression matching:

`WHERE something = "testing \"something\""`

`WHERE something LIKE "%testing \"something\""`